packages/material-ui/src/FormControl/FormControl.d.ts

```diff
@@ -67,6 +67,11 @@ export interface FormControlTypeMap<P = {}, D extends React.ElementType = 'div'>
      * @default 'medium'
      */
     size?: OverridableStringUnion<'small' | 'medium', FormControlPropsSizeOverrides>;
+    /**
+     * The width of the start adornment in pixels.
```

Comment:
A "pixels" can have different interpretations. Best to use the CSS unit.

```suggestion
     * The width of the start adornment in `px`.
```